src/languages/marko.coffee

```diff
@@ -1,8 +1,15 @@
+# Get Atom defaults
+scope = ['text.marko']
+tabLength = atom?.config.get('editor.tabLength', scope: scope) ? 4
+softTabs = atom?.config.get('editor.softTabs', scope: scope) ? true
+defaultIndentSize = (if softTabs then tabLength else 4)
+defaultIndentChar = (if softTabs then " " else "\t")
+defaultIndentWithTabs = not softTabs
+
 module.exports = {
 
   name: "Marko"
   namespace: "marko"
-  fallback: ['html']
```

Comment:
> Marko templates no longer use the HTML syntax (Marko uses a variation of HTML that is not always valid HTML). Therefore, you would not be able to use the default HTML beautifier to beautify a Marko v3 template. 

That's not what `fallback` does 😃. `fallback` is only related to how to get configuration settings.

> Also, currently the Marko beautifier doesn't support the majority of the settings that the HTML beautifier supports. That's my thought process for removing html as a fallback,

Leaving `html` as a fallback means that a user could set `html.indent_size` and if `marko.indent_size` is not set then it will look at `html.indent_size` next, etc. So the options `indent_size` and `indent_char` would be applicable, however `syntax` would not.

Fallback option was essentially a way to allow users to configure for one main language (`html`) and similar languages (`marko`?) could then take advantage of those settings without requiring explicitly changing the settings for that language itself (`marko.indent_size` directory).